client-v2/src/components/FrontsEdit/FrontsMenu.tsx

```diff
@@ -160,6 +183,8 @@ class FrontsMenu extends React.Component<Props, State> {
               </FrontsMenuSubHeading>
               <FrontsList
                 onSelect={this.onSelectFront}
+                onStar={this.onFavouriteFront}
```

Comment:
should we consistently use favourite or starred everywhere?

Reply:
Sorry - missed this in my regex search to rename variables. Will fix all before merging.